app/models/position.rb

```diff
@@ -18,13 +18,8 @@ def name_and_department
     "#{name} (#{department.name})"
   end
 
-  def default_not_hiring_text
-    "We are not currently hiring for #{name}. Please check back."
-  end
-
-  def configured_not_hiring_text
-    configured_value([:deactivated_application,
-                      yamlize(department.name), yamlize(name)],
-                     default: default_not_hiring_text)
+  def not_hiring
+    not_hiring_text ||
+      "We are not currently hiring for #{name}. Please check back."
```

Comment:
What would you prefer instead?

On Wed, Jul 6, 2016 at 4:57 PM, David Faulkenberry <notifications@github.com
> wrote:

> In app/models/position.rb
> <https://github.com/umts/jobapps/pull/259#discussion_r69810028>:
>
> > @@ -18,13 +18,8 @@ def name_and_department
> >      "#{name} (#{department.name})"
> >    end
> >
> > -  def default_not_hiring_text
> > -    "We are not currently hiring for #{name}. Please check back."
> > -  end
> > -
> > -  def configured_not_hiring_text
> > -    configured_value([:deactivated_application,
> > -                      yamlize(department.name), yamlize(name)],
> > -                     default: default_not_hiring_text)
> > +  def not_hiring
> > +    not_hiring_text ||
> > +      "We are not currently hiring for #{name}. Please check back."
>
> I don't remember that, but I know I've said this before about those words.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/umts/jobapps/pull/259/files/6ca34c6f977ccc5ed6db58976659693bd159e804#r69810028>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJMtBPRxINjaYDrUWn8lscJaHC1XoKLqks5qTBaogaJpZM4JGOXR>
> .
>